distributed/dashboard/components/Untitled.ipynb

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Is this file intended to be here or pushed accidentally?

Reply:
accident -- sorry